src/main/java/org/folio/circulation/rules/Drools.java

```diff
@@ -107,11 +120,11 @@ public JsonArray loanPolicies(MultiMap params, Location location) {
    * @param location - location with institution, library and campus
    * @return the name of the request policy
    */
-  public String requestPolicy(MultiMap params, Location location) {
+  public Pair<String, List<String>> requestPolicy(MultiMap params, Location location) {
     KieSession kieSession = createSession(params, location);
     kieSession.fireAllRules();
     kieSession.dispose();
-    return match.requestPolicyId;
+    return new ImmutablePair<>(match.requestPolicyId, new ArrayList<>());
```

Comment:
It is probably out of scope, but we need further make it consistent for all the policy types or make changes in the class hierarchy, so the `conditions` list will be returned where it is really needed.

Does it make sense?